docs/source/Makefile

```diff
@@ -0,0 +1,192 @@
+# Makefile for Sphinx documentation
```

Comment:
can you add a comment about where you pulled this from?  I doubt you wrote all this yourself, right?